### ISSUE TITLE
clientconfig: Add support for SSL setting to disable verification

### ISSUE THREAD

**Issue by haxorof** [gophercloud/utils#45]
In the OpenStack documentation there are some settings which can be made to disable certificate verification. https://docs.openstack.org/os-client-config/latest/user/configuration.html#ssl-settings

It would be good to start with the disable verification switch which is same as setting environment variable OS_INSECURE.

Example below:
```
# clouds.yaml 
clouds:
  insecure: 
    auth: 
      ...
      verify: False
```

This can later be used in Terraform to avoid specifying the insecure config on the OpenStack provider

Cheers!

**Comment by jtopjian** [comment]
If it's a valid `clouds.yaml` feature, then it sounds good to me :)

Two things to note for whoever wants to add this:

1. Similar to https://github.com/gophercloud/gophercloud, we'll need to reference where in the Python libraries this is defined. I've found hunting for clientconfig stuff more difficult than other OpenStack features. There are 3-4 different libraries to look at (os-client-config, osc-lib, openstacksdk, python-openstackclient).

2. This feature should _not_ do any type of http client configuration. That needs to be done at the application-level. Using Terraform as the example, Terraform will still do the TLS configuration [here](https://github.com/terraform-providers/terraform-provider-openstack/blob/master/openstack/config.go#L116-L149) but it'll check the `clouds.yaml` configuration (in addition to the Terraform schema).

**Comment by haxorof** [comment]
Python reference:
https://github.com/openstack/os-client-config/blob/stable/queens/os_client_config/cloud_config.py#L114

**Comment by jtopjian** [comment]
Implemented in #50
